core/string/sub_spec.rb

```diff
@@ -396,7 +396,7 @@
   end
 
   it "ignores non-String keys" do
-    "hello".sub(/(ll)/, 'll' => 'r', :ll => 'z').should == "hero"
+    "hello".sub(/(ll)/, 'll' => 'r', ll: 'z').should == "hero"
```

Comment:
The "non-string' keys in the test description and the very similar looking ell's encouraged this.  Perhaps the string itself should be changed for clarity?